docs/en/latest/references/apisix_route_v2.md

```diff
@@ -21,83 +26,85 @@ title: ApisixRoute/v2 Reference
 #
 -->
 
+See [concepts](https://apisix.apache.org/docs/ingress-controller/concepts/apisix_route) to learn more about how to use the ApisixRoute resource.
+
 ## Spec
 
-Meaning of each field in the spec of ApisixRoute are followed, the top level fields (`apiVersion`, `kind` and `metadata`) are same as other Kubernetes Resources.
+See the [definition](https://github.com/apache/apisix-ingress-controller/blob/master/samples/deploy/crd/v1/ApisixRoute.yaml) on GitHub.
+
+The table below describes each of the attributes in the spec. The fields `apiVersion`, `kind`, and `metadata` are similar to other Kubernetes resources and are excluded below.
+
+| Attribute                            | Type               | Description                                                                                                                                                                               |
+| ------------------------------------ | ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| http                                 | array              | HTTP Route rules.                                                                                                                                                                         |
+| http[].name                          | string (required)  | Route rule name.                                                                                                                                                                          |
+| http[].priority                      | integer            | Route priority. Used to determined which Route to use when multiple routes contain the same URI. Large number means higher priority.                                                      |
+| http[].timeout                       | object             | Timeouts for connecting, sending, and receiving messages between Ingress and the service. This will overwrite the value configured in `ApisixUpstream` resource.                          |
+| http[].timeout.connect               | string             | Connect timeout in the form "72h3m0.5s".                                                                                                                                                  |
```

Comment:
Hi @navendu-pottekkat, is this expected?

<img width="499" alt="image" src="https://user-images.githubusercontent.com/2106987/199390099-3c0c7919-6a28-480c-a30e-f7763c04b3ae.png">

Reply:
@juzhiyuan I did not understand your question.

Reply:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/2106987/199407627-c1137ba5-0e66-4354-af74-26d84581be9b.png">

This form is valid?

Reply:
I don't know. I did not change it. @tao12345666333 could you confirm?

Reply:
let me check it.